src/hotspot/share/gc/z/zReferenceProcessor.hpp

```diff
@@ -69,7 +69,9 @@ class ZReferenceProcessor : public ReferenceDiscoverer {
 public:
   ZReferenceProcessor(ZWorkers* workers);
 
-  void set_soft_reference_policy(bool clear);
+  void set_soft_reference_policy(bool clear_all_soft_references);
+  bool uses_clear_all_soft_reference_policy() const;
```

Comment:
The method name should be `uses_clear_all_soft_references_policy`. Please note the letter `s` after the `reference`.